<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaitingListView</class>
 <widget class="QMainWindow" name="WaitingListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WaitingListView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="headerHLayout" stretch="0,1,0,0">
      <item>
       <widget class="QLabel" name="headerLbl">
        <property name="font">
         <font>
          <family>Berlin Sans FB</family>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vaccination Waiting List</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="indent">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="headerSpacer_MH">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../WidgetsStack.qrc">
          <normaloff>:/WidgetsStack/Assets/back-button.png</normaloff>:/WidgetsStack/Assets/back-button.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="headerSpacer_RH">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="middleHLayout" stretch="3,0,0,0,3">
      <item>
       <layout class="QVBoxLayout" name="dose1VLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="dose1HeaderLbl">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>30</pointsize>
             </font>
            </property>
            <property name="text">
             <string>1x Dose</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dose1CountLbl">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>x100 user</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="dose1Table">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableWidget {
alternate-background-color: rgb(102, 147, 206);
}


QHeaderView::section { 
color:white; 
background-color:#297EB7; 
}</string>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>70</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>170</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>National ID</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Full Name </string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gender</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Age</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Country</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Government</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dose1NavHLayout" stretch="4,2,2,2,2,2">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="dose1PreviousBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../WidgetsStack.qrc">
              <normaloff>:/WidgetsStack/Assets/previous-icon.png</normaloff>:/WidgetsStack/Assets/previous-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dose1CurrentTabLbl">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>40</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dose1NextBtn">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../WidgetsStack.qrc">
              <normaloff>:/WidgetsStack/Assets/next-icon - Copy.png</normaloff>:/WidgetsStack/Assets/next-icon - Copy.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="dose1FinishBtn">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../WidgetsStack.qrc">
              <normaloff>:/WidgetsStack/Assets/correct-icon.png</normaloff>:/WidgetsStack/Assets/correct-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="dose2VLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="dose2HeaderLbl">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>30</pointsize>
             </font>
            </property>
            <property name="text">
             <string>2x Doses</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="dose2CountLbl">
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>x100 user</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="dose2Table">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableWidget {
alternate-background-color: rgb(102, 147, 206);
}


QHeaderView::section { 
color:white; 
background-color:#297EB7; 
}</string>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>70</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>170</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>National ID</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Full Name </string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gender</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Age</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Country</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Government</string>
           </property>
           <property name="font">
            <font>
             <family>Berlin Sans FB</family>
             <pointsize>16</pointsize>
            </font>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dose2NavHLayout" stretch="4,2,2,2,2,2">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="dose2PreviousBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../WidgetsStack.qrc">
              <normaloff>:/WidgetsStack/Assets/previous-icon.png</normaloff>:/WidgetsStack/Assets/previous-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dose2CurrentTabLbl">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>40</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dose2NextBtn">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../WidgetsStack.qrc">
              <normaloff>:/WidgetsStack/Assets/next-icon - Copy.png</normaloff>:/WidgetsStack/Assets/next-icon - Copy.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="dose2FinishBtn">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../WidgetsStack.qrc">
              <normaloff>:/WidgetsStack/Assets/correct-icon.png</normaloff>:/WidgetsStack/Assets/correct-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../WidgetsStack.qrc"/>
 </resources>
 <connections/>
</ui>
